providers/google/tests/provider_tests/google/cloud/hooks/test_cloud_sql.py

```diff
@@ -1596,6 +1596,8 @@ def get_processor():
     processor = os.uname().machine
     if processor == "x86_64":
         processor = "amd64"
+    if processor == "aarch64":
```

Comment:
One of the tests in google was failing on MacOS because we missed the mapping here.